test/refactor_nrepl/analyzer_test.clj

```diff
@@ -0,0 +1,10 @@
+(ns refactor-nrepl.analyzer-test
+  (:require
+   [clojure.java.io :as io]
+   [refactor-nrepl.analyzer :as sut]
+   [clojure.test :refer [deftest is]]))
+
+(deftest ns-ast-test
```

Comment:
I verified that if we comment out this PR's main change, this test would fail, complaining about `go-loop`